src/Sylius/Bundle/AdminBundle/Resources/views/Country/Component/countryType.html.twig

```diff
@@ -0,0 +1,16 @@
+{% form_theme form '@SyliusAdmin/Shared/form_theme.html.twig' %}
+
+{% set main_event = twig_event_name(attributes.all.main_event, 'component_form') %}
+{% set fallback_event = twig_event_name(attributes.all.fallback_event, 'component_form') %}
```

Comment:
```suggestion
{% set main_event = twig_event_name(_context.main_event, 'component_form') %}
{% set fallback_event = twig_event_name(_context.fallback_event, 'component_form') %}
```

From what I see, you passed `main_event` and `fallback_event` along with the twig event, so they're accessible directly by using `main_event` or `_context.main_event`. I prefer the second option, to avoid "magic".

Reply:
I have added two LiveProp properties,  `mainEvent` and `fallbackEvent` to the Components, and I have utilized `this.mainEvent`.